src/components/DocumentRenderer/WebViewFrame.tsx

```diff
@@ -64,9 +64,14 @@ export const WebViewFrame: FunctionComponent<WebViewFrame> = ({
           const templates = window.openAttestation({type: "GET_TEMPLATES", payload: documentToRender});
           window.ReactNativeWebView.postMessage(JSON.stringify(templates));
 
-          // Add spacing container to the bottom of the document so that it isn't blocked by the bottom sheet
+          // Add spacing container to ensure the document isn't blocked by the bottom sheet
           const spacingContainer = document.createElement('div');
-          spacingContainer.style.height = '30vh';
+          spacingContainer.style.position = 'absolute';
+          spacingContainer.style.top = 0;
+          spacingContainer.style.left = 0;
+          spacingContainer.style.right = 0;
+          spacingContainer.style.zIndex = -99;
```

Comment:
![image](https://user-images.githubusercontent.com/7406870/73324875-750c9b00-4287-11ea-9eae-c3491ced3750.png)

I'm not a expert at this, but google says minimum z-index is 0....

Reply:
As discussed, z-index can be negative since browsers resolve the stacking order by looking at the relative differences in z-index. I've added a `pointer-events: none` to be sure that even if it blocks some elements, all clicks will passthrough.